jellyfish.livemd

```diff
@@ -81,41 +77,105 @@ Add Jellyfish SDK to Elixir deps in `mix.exs`:
-5. From the above callback push the token over socket
-6. We should be handling termination of the room - we will handle it better later, for now we may add a simple process monitoring channel's process calling `Room.delete` on channel exit.
+## Client to backend connection
+
+We'll start by setting up our frontend to backend connection and calling Jellyfish API. 
+Our goal will be to obtain a token associated with a room and peer that frontend can use to establish a direct connection to Jellyfish.
+Later on, we'll build architecture managing the rooms and peers on our backend, but we'll start with something much simpler, to be precise each will create their own room and join it.
+
+1. For authorization in jellyfish, we'll need to set up the same secret/token as passed to the Jellyfish. We can do this by setting in `config.exs` variable `:server_api_token` in application `:jellyfish_server_sdk`, but we can also pass in the construction of sdk client.
+2. Create a new [`Phoenix.Channel`](https://hexdocs.pm/phoenix/channels.html), which will be responsible for the communication browser and backend.
+3. On join, send a message to `self` and let the user join.
+4. Handle the message sent above and create a new [`Jellyfish.Client`](https://hexdocs.pm/jellyfish_server_sdk/Jellyfish.Client.html) and use it to create a new [`Jellyfish.Room`](https://hexdocs.pm/jellyfish_server_sdk/Jellyfish.Room.html). Then add a new `webtrc` peer to the jellyfish room. Remember to assign `client`, `room_id`, and `peer_id` to the socket.
+5. From the above callback push the `PeerToken` over the socket.
+6. We should be handling the termination of the room - we will handle it better later, for now, we may simply call `Room.delete` on the channel exit.
 7. Go to the JS part of the channel and handle the token message.
 
+If you correctly implement all points above you should receive a token on fronted and could log it.
+
 ## Frontend - let's send some media to Jellyfish
 
-The next step is to use the obtained token to start sending media to Jellyfish. We're going to use JS WebRTC APIs and [Jellyfish TS client](https://github.com/jellyfish-dev/jellyfish-react-client/blob/update-readme/src/jellyfish/readme.md).
+The current state is that on jellyfish is created room and peer and also the client (browser) received a `PeerToken`.
+The next step is to use this token to successfully connect to Jellyfish and start sending media to it. 
+Then we will have to add handling for messages received from Jellyfish. 
+We're going to use JS WebRTC APIs and [Jellyfish TS client](https://jellyfish-dev.github.io/ts-client-sdk/). 
+This client SDK notifies the browser about situations like a new peer joining or receiving a new track.
+
+First things first we have to initialize our jellyfish TS client, it will interact with your js app, through callbacks, that's why implementing them will be our next task.
```

Comment:
```suggestion
First things first. We have to initialize our jellyfish TS client, it will interact with your js app, through callbacks, that's why implementing them will be our next task.
```